### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/mermaid.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#603] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/mermaid.html

让你的 VuePress 站点中的 Markdown 文件支持 mermaid (https://mermaid.js.org/)。 配置 tab TS tab JS 语法 除了使用 mermaid 代码块，你也可以直接使用以下代码块： class: classDiagram; c4c: C4Context; er: erDiagram; gantt:...

https://plugin-md-enhance.vuejs.press/zh/guide/mermaid.html

**Comment by lingdu990130556** [comment]
思维导图本地显示正常，打包后显示挤在一起为什么

**Reply by lingdu990130556** [reply]
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/76582458/17107fbf-ba41-4786-afd1-30d4c09d29c4)
![image](https://github.com/vuepress-theme-hope/giscus-discussions/assets/76582458/4fe7254f-04a8-47d5-a898-43067277a5da)
